src/backends/github.rs

```diff
@@ -340,9 +346,10 @@ impl UpdateBuilder {
 
     /// Set the exe's name. Also sets `bin_path_in_archive` if it hasn't already been set.
     pub fn bin_name(&mut self, name: &str) -> &mut Self {
-        self.bin_name = Some(name.to_owned());
+        let raw_bin_name = format!("{}{}", name, EXECUTABLE_SUFFIX);
+        self.bin_name = Some(raw_bin_name.to_owned());
```

Comment:
No need to convert `raw_bin_name` using `to_owned()` since it's already a `String`.  Since `raw_bin_name` is used on the next line I'd say a `.clone()` would show the intentions better.